data/mods/gen5/moves.js

```diff
@@ -266,6 +266,23 @@ let BattleMovedex = {
 	},
 	electroball: {
 		inherit: true,
+		basePowerCallback(pokemon, target) {
+			let ratio = (pokemon.getStat('spe') / Math.max(1, target.getStat('spe')));
+			this.debug([40, 60, 80, 120, 150][(Math.floor(ratio) > 4 ? 4 : Math.floor(ratio))] + ' bp');
```

Comment:
???

why not just return this instead of using 12 lines of if blocks to do the same thing?

Reply:
Because that's how the existing copy of that code in the current-gen section is structured.

Reply:
RIP. Can we please modify it in both places then? The `if` blocks are really ugh...